serverless/src/layer.ts

```diff
@@ -40,6 +40,7 @@ const runtimeToLayerName: { [key: string]: string } = {
   "python3.6": "Datadog-Python36",
   "python3.7": "Datadog-Python37",
   "python3.8": "Datadog-Python38",
+  LambdaExtension: "Datadog-Extension",
```

Comment:
technically I dont think the extension is a runtime but this was the easiest way I saw of adding support for the extension without making too much code changes.

Reply:
I would find some way to change this -- I do think it's confusing to have "LambdaExtension" as a runtime -- maybe we can break it out into a separate constant somehow. Even if we need to make a code change, I think that would still create a cleaner, easier-to-understand solution.

Reply:
@nhinsch Just moved it to a constant